app/attachments.js

```diff
@@ -0,0 +1,99 @@
+const crypto = require('crypto');
+const fse = require('fs-extra');
+const isArrayBuffer = require('lodash/isArrayBuffer');
+const isString = require('lodash/isString');
+const path = require('path');
+const toArrayBuffer = require('to-arraybuffer');
+
+
+const PATH = 'attachments.noindex';
+
+//      getPath :: AbsolutePath -> AbsolutePath
+exports.getPath = (userDataPath) => {
+  if (!isString(userDataPath)) {
+    throw new TypeError('`userDataPath` must be a string');
```

Comment:
Can you clarify what the particular concern is? The rationale is to use the Markdown syntax we’re so used to as developers for denoting identifiers.

Reply:
Backticks are harder for me to scan and understand. It's a twitch thing, and makes the code take more time to parse. IMO, markdown syntax is only useful when it gets converted to markdown.

I'm not really worried about a confusing error message like 'data must be a string' because we'll have the stack and therefore line numbers to help us out with addressing the error. If we're really looking for clarity, we might want to put the name of the method in the error.